source/agora/consensus/EnrollmentManager.d

```diff
@@ -898,41 +891,37 @@ unittest
 
     Height height = Height(2);
 
-    Enrollment[] enrollments;
-    foreach (idx, kp; pairs[0 .. 3])
-    {
-        enrollments ~= EnrollmentManager.makeEnrollment(
-            utxo_hashes[idx], kp, height, params.ValidatorCycle);
-    }
-
```

Comment:
Ditto.

Reply:
Those are both required to make the tests pass with this change.